.github/workflows/BFS.yaml

```diff
@@ -0,0 +1,235 @@
+# Copyright (c) 2023, Oracle Corporation and/or its affiliates.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at
+# http://oss.oracle.com/licenses/upl.
+
+# ---------------------------------------------------------------------------
+# Verrazzano GitHub Actions Keycloak BFS.
+# ---------------------------------------------------------------------------
+
+name: Keycloak BFS
+run-name: GitHub Actions workflow triggered by changes to Keycloak BFS 🔑
+
+on:
+  push:
+    branches:
+      - 'oracle/release/*'
+      - 'bfs/*'
+  workflow_dispatch:
+
+env:
+  REGISTRY: ghcr.io
+  NAMESPACE: verrazzano
+
+concurrency:
+  # Only run once for latest commit per ref and cancel other (previous) runs.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+      # This is used to complete the identity challenge
+      # with sigstore/fulcio when running outside of PRs.
+      id-token: write
+
+    steps:
+      # Get Keycloak version
+      - name: Get Keycloak version
+        run: echo "KEYCLOAK_VERSION=$(echo ${{ github.ref_name }} | sed 's:.*/::')" >> "$GITHUB_ENV"
+
+      # Checkout keycloak repo
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          ref: oracle/release/${{ env.KEYCLOAK_VERSION }}
```

Comment:
The branch here is the oracle/release/* branch but above we are also triggering when folks are using bfs/* branches. Seems like this won't work for BFS branches, but only for the oracle/release ones?

Reply:
Get rid of ref